src/components/headers/collection-header.tsx

```diff
@@ -190,16 +191,15 @@ export class CollectionHeader extends React.Component<IProps, IState> {
             </Tooltip>
           ),
       this.context.user.model_permissions.delete_collection && (
-        <div data-cy='delete-version-dropdown'>
-          <DropdownItem
-            key='2'
-            onClick={() =>
-              this.openDeleteModalWithConfirm(collection.latest_version.version)
-            }
-          >
-            {t`Delete version ${collection.latest_version.version}`}
-          </DropdownItem>
-        </div>
+        <DropdownItem
+          data-cy='delete-version-dropdown'
```

Comment:
the same  data-cy value is already used in the first DropdownItem here

```suggestion
          data-cy='delete-version-version'
```